testing/opae-cxx/test_properties_cxx_core.cpp

```diff
@@ -286,4 +286,4 @@ TEST_P(properties_skx_dcp_cxx_core, get_guid_valid) {
 }
 
 INSTANTIATE_TEST_CASE_P(properties_skx_dcp, properties_skx_dcp_cxx_core,
```

Comment:
Since you're calling out the Intel driver here, should the fixture name reflect that as well?